template/logentry/template.proto

```diff
@@ -33,4 +33,10 @@ message Template {
 
     // Severity indicates the importance of the log entry.
     string severity = 3;
+
+    // An expression to compute the type of the monitored resource this log entry is being recorded on.
+    string monitored_resource_type = 4;
```

Comment:
call the field monitored_resource to be generic?

Reply:
I'm not sure what you mean by generic. Today we only have a single example of monitored resources, which are in Stackdriver. Stackdriver represents a MR as a type + labels. If we call the field only "monitored_resource" then I don't think it's clear what I should populate the field with.

Reply:
My understanding is the template is meant to be general purpose. So adapter can share with it. In stackdriver, it is a type + labels, in other monitor BE, could be just a url + key value pair attribute. My thought is name it resouce + dimension and let adapter developer to define its exact meaning. Not a big thing, so it is up to you.